beeline-spring-boot-starter/pom.xml

```diff
@@ -16,7 +16,7 @@
     <description>Spring Boot Starter module to auto-configure Spring Boot with the Honeycomb Beeline for Java</description>
 
     <properties>
-        <beelineVersion>1.0.8</beelineVersion>
+        <beelineVersion>1.1.1</beelineVersion>
```

Comment:
Let's keep this at 1.1.0

Reply:
As discussed in slack, I have not reverted this change.